### ISSUE TITLE
Upgrading to bs-platform@7.0.0 errors

### ISSUE THREAD

**Issue by simontegg** [apollographql/reason-apollo#215]
Upgrading my project to bucklescript 7 gives the following build error. 

```

 We've found a bug for you!
  /home/......node_modules/reason-apollo/src/graphql-types/ReasonApolloQuery.re 185:7-15
  
  183 ┆ <JsQuery
  184 ┆   query=graphqlQueryAST
  185 ┆   variables 
  186 ┆   pollInterval
  187 ┆   notifyOnNetworkStatusChange
  

  [variables] has type:
    option(Js.Json.t)
  But somewhere wanted:
    Js.Json.t (defined as Js.Json.t)

```

**Comment by MargaretKrutikova** [comment]
This seems to be resolved with [this PR](https://github.com/apollographql/reason-apollo/pull/213), but not released yet, is there any way to release a new npm version, @Gregoirevda ? 

`apollo-reason-hooks` also depends on this package and we can't upgrade to `bs-platform` 7 without `reason-apollo` being compatible with it.

**Comment by Gregoirevda** [comment]
@MargaretKrutikova the merged PR has been published under `0.18.0`

**Comment by simontegg** [comment]
Thanks for the quick response. That did it.
